File: src/main/java/net/ndrei/teslacorelib/compatibility/FontRendererUtil.java
package net.ndrei.teslacorelib.compatibility;

import net.minecraft.client.Minecraft;
import net.minecraft.client.gui.FontRenderer;
import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

/**
 * Created by CF on 2017-02-09.
 */
public final class FontRendererUtil {
    @SideOnly(Side.CLIENT)
    public static FontRenderer getFontRenderer() {
        return Minecraft.getMinecraft().fontRenderer;
    }
}

File: src/main/java/net/ndrei/teslacorelib/compatibility/FontRendererUtil.kt
package net.ndrei.teslacorelib.compatibility

import net.minecraft.client.Minecraft
import net.minecraft.client.gui.FontRenderer
import net.minecraftforge.fml.relauncher.Side
import net.minecraftforge.fml.relauncher.SideOnly

/**
 * Created by CF on 2017-06-28.
 */
@SideOnly(Side.CLIENT)
object FontRendererUtil {
    val fontRenderer: FontRenderer
        get() = Minecraft.getMinecraft().fontRenderer
}